src/token_contract/src/test/main.test.ts

```diff
@@ -27,15 +27,18 @@ import {
 import { EscrowContract, EscrowContractArtifact } from '@aztec/noir-contracts.js/Escrow';
 import { ContractInstanceDeployerContract } from '@aztec/noir-contracts.js/ContractInstanceDeployer';
 
-const startPXE = async (id: number = 0) => {
-  const { PXE_URL = `http://localhost:${8080 + id}` } = process.env;
-  const pxe = createPXEClient(PXE_URL);
+const createPXE = async (id: number = 0) => {
+  // todo: we should probably define testing fixtures for this kind of configuration
```

Comment:
let's avoid having `todo` in small caps, since it can get lost in a `TODO` search